test/executables/react16_ci_precheck_test.dart

```diff
@@ -160,23 +197,13 @@ ProcessResult runCiPrecheck({String onDirectory}) {
   return result;
 }
 
-ProcessResult runCiPrecheckWithFakeDartProject(
```

Comment:
Inlined this so I wouldn't have to change `pubspecCreator` to a list another time haha